pkg/config/config.go

```diff
@@ -126,7 +179,31 @@ func InitCliConfig(configAndStacksInfo schema.ConfigAndStacksInfo, processStacks
 	}
 
 	if !configFound {
-		return cliConfig, NotFound
+		// If `atmos.yaml` not found, use the default config
+		// Set `ATMOS_LOGS_LEVEL` ENV var to "Debug" to see the message about Atmos using the default CLI config
+		logsLevelEnvVar := os.Getenv("ATMOS_LOGS_LEVEL")
+		if logsLevelEnvVar == "Debug" {
+			u.PrintMessageInColor("'atmos.yaml' CLI config was not found in any of the searched paths: system dir, home dir, current dir, ENV vars.\n"+
+				"Refer to https://atmos.tools/cli/configuration for details on how to configure 'atmos.yaml'.\n"+
+				"Using the default CLI config:\n", color.New(color.FgCyan))
+
+			err = u.PrintAsYAML(defaultCliConfig)
+			if err != nil {
+				return cliConfig, err
+			}
+			fmt.Println()
+		}
+
+		j, err := json.Marshal(defaultCliConfig)
```

Comment:
My concern is we don't provide a way to not use the `defaultCliConfig`. If our CLI config ships some setting to users using their own `atmos.yaml` config, it can alter the behavior. I think the default behavior should be exactly what you have here, but some mechanism to disable the default config from getting merged.

I was thinking it could be something like:

### Disable, then enable defaults (only to show how it works)
```
defaults: false
import:
- @defaults
``` 

### Which is identical to
```
defaults: true
```

By default, I think `defaults: true` should be set if `defaults` is not present in the discovered local atmos config file.

Reply:
I'd like to be able to do something like:
```
import:
- @defaults
- @cloudposse/defaults
- ~/.atmos.d/**.yaml
```

Which would add a bunch of defaults we can use in our repos. For example, in `cloudposse/terraform-aws-vpc`, we define a `.atmos.yml` which has:

```
import:
- @defaults
- @cloudposse/defaults
```

And now we can run the following, calling on a custom configuration by cloud posse.
```
atmos generate readme
```

Reply:
Agree